Failure causes artifacts to not upload

During a load test I had some HTTP requests that failed due to the remote system unable to handle the load.  Once the test completed the "Publish Load Testing Results" step shows the error "##[error]There are one or more test failures detected in result files. Detailed summary of published test results can be viewed in the Tests tab." which is expected since some http requests failed.  The issue is the following step which should upload the jMeter results (RESULTS: Publish Load Test Artifacts) does not run due to the prior step's error.    This means there is no way to see the actual jMeter dashboard when any failure occurs.

--- a/pipelines/azure-pipelines.load-test.yml
+++ b/pipelines/azure-pipelines.load-test.yml
@@ -103,6 +103,7 @@ steps:
 
 - publish: $(JMETER_DIRECTORY_OUTPUT)
   artifact: JMeterResults
+  condition: succeededOrFailed()
   displayName: 'RESULTS: Publish Load Test Artifacts'
 
 - script: terraform destroy -auto-approve

